src/gucs.rs

```diff
@@ -0,0 +1,42 @@
+use pgrx::*;
+use std::ffi::CStr;
+
+pub static NEON_AUTH_KID_RUNTIME_PARAM: &str = "neon.auth.kid";
+pub static NEON_AUTH_KID: GucSetting<i32> = GucSetting::<i32>::new(0);
```

Comment:
GUC doesn't support `i64`, later `i32` is casted into `i64` for backwards compatibility

Reply:
In a pass I removed this GUC and folded it into the JWK